README.rst

```diff
@@ -260,11 +260,3 @@ make install
 
 The default installation prefix is /usr/local.  You may need to run
 'ldconfig' after performing a 'make install'.
-
```

Comment:
@bdpayne This was before my time, I have no idea if people are getting here by coming from XenAccess.  Maybe mention it in a new **History** section of the README?